packages/invoices/src/api.ts

```diff
@@ -67,6 +71,14 @@ const linesGuard = (invoiceLines: InvoiceLineConnector | ConnectorError) => {
   throw new Error('Unknown');
 };
 
+const stockLineGuard = (stockLine: StockLineResponse): StockLineNode => {
+  if (stockLine.__typename === 'StockLineNode') {
+    return stockLine;
+  }
+
+  throw new Error('Unknown');
+};
```

Comment:
these are getting very tedious 🤔 

I wonder if we need to speak to the team about it? I supposes it's fine, to write this small function but if we query for an invoice line with it's stock line and we get a response that there was an error fetching the stock line, I don't know what else we would do other than say the invoice is broken, you can't edit it sorry 🤔

Reply:
yeah, I'm wondering what we can do about it - they are tedious to write and have to be so specific; would be nice if the response had an error of a standard shape that could be checked